src/uu/tty/src/tty.rs

```diff
@@ -12,68 +12,65 @@
 #[macro_use]
 extern crate uucore;
 
+use clap::{App, Arg};
 use std::ffi::CStr;
 use uucore::fs::is_stdin_interactive;
 
 extern "C" {
     fn ttyname(filedesc: libc::c_int) -> *const libc::c_char;
 }
 
-static NAME: &str = "tty";
 static VERSION: &str = env!("CARGO_PKG_VERSION");
+static ABOUT: &str = "Print the file name of the terminal connected to standard input.";
 
-pub fn uumain(args: impl uucore::Args) -> i32 {
-    let args = args.collect_str();
-
-    let mut opts = getopts::Options::new();
+mod options {
+    pub const SILENT: &str = "silent";
+}
 
-    opts.optflag("s", "silent", "print nothing, only return an exit status");
-    opts.optflag("h", "help", "display this help and exit");
-    opts.optflag("V", "version", "output version information and exit");
+fn get_usage() -> String {
+    format!("{0} [OPTION]...", executable!())
+}
 
-    let matches = match opts.parse(&args[1..]) {
-        Ok(m) => m,
-        Err(f) => crash!(2, "{}", f),
-    };
+pub fn uumain(args: impl uucore::Args) -> i32 {
+    let args = args.collect_str();
+    let usage = get_usage();
 
-    if matches.opt_present("help") {
-        println!("{} {}", NAME, VERSION);
-        println!();
-        println!("Usage:");
-        println!("  {} [OPTION]...", NAME);
-        println!();
-        print!(
-            "{}",
-            opts.usage("Print the file name of the terminal connected to standard input.")
-        );
-    } else if matches.opt_present("version") {
-        println!("{} {}", NAME, VERSION);
-    } else {
-        let silent = matches.opt_present("s");
+    let matches = App::new(executable!())
+        .version(VERSION)
+        .about(ABOUT)
+        .usage(&usage[..])
+        .arg(
+            Arg::with_name(options::SILENT)
+                .long(options::SILENT)
+                .alias("quiet")
```

Comment:
use `visible_alias` to make it show up in the help text

Reply:
Oh nice, I didn't know that.